src/sources/models.js

```diff
@@ -302,6 +303,47 @@ class NarrativeSubresource extends Subresource {
 }
 
 
+// TODO: rename to GroupOverview if this will only be used by groups? If so:
+// 1. Change the name of sources.groups.GroupOverview
+// 2. Change the name of authz.tags.Type.Overview
+class Overview extends Resource {
```

Comment:
https://github.com/nextstrain/nextstrain.org/labels/request%20for%20comments

Reply:
This is currently specific to groups with no plans/vision for extending it, so my preference is to keep it tightly scoped.